manta-pay/abstract-protocol.tex

```diff
@@ -87,29 +175,38 @@ \subsection{Abstract Cryptographic Schemes}
         \item Encryption: Given an ephemeral secret key $\esk: \KA.\SecretKey$, a public key $\pk: \KA.\PublicKey$, and plaintext $p : \SYM.\Plaintext$, we produce the pair
             \[m : \KA.\PublicKey \times \SYM.\Ciphertext := \left(\KA.\derive(\esk), \SYM.\encrypt(\KDF(\KA.\agree(\esk, \pk)), p)\right)\]
         \item Decryption: Given a secret key $\sk : \KA.\SecretKey$, and an encrypted message, as above, $m := (\epk, c)$, we can decrypt $m$, producing the plaintext,
-            \[p : \Option(\SYM.\Plaintext) := \SYM.\decrypt(\KDF(\KA.\agree(\sk, \epk)), c)\]
+            \[p : \Option\bra{\SYM.\Plaintext} := \SYM.\decrypt(\KDF(\KA.\agree(\sk, \epk)), c)\]
             which should decrypt successfully if the $\KA.\PublicKey$ that $m$ was encrypted with is the derived key of $\sk : \KA.\SecretKey$.
     \end{itemize}
 
-    \textbf{Notation}: We denote the above \emph{encrypted message} type as $\Message := \SYM.\Ciphertext \times \KA.\PublicKey$, and the above two algorithms by
+    \textbf{Notation}: We denote the above \emph{encrypted message} type as $\Encrypted\bra{\SYM.\Plaintext} := \KA.\PublicKey \times \SYM.\Ciphertext$, and the above two algorithms by
     \begin{align*}
-        \encrypt &: \KA.\SecretKey \times \KA.\PublicKey \times \SYM.\Plaintext \to \Message \\
-        \decrypt &: \KA.\SecretKey \times \KA.\PublicKey \times \SYM.\Ciphertext \to \Option(\SYM.\Plaintext)
+        \encrypt &: \KA.\SecretKey \times \KA.\PublicKey \times \SYM.\Plaintext \to \Encrypted\bra{\SYM.\Plaintext} \\
+        \decrypt &: \KA.\SecretKey \times \KA.\PublicKey \times \SYM.\Ciphertext \to \Option\bra{\SYM.\Plaintext}
     \end{align*}
     \textbf{Security Properties}: The HPKE constructed from $\KA$, $\KDF$, and $\SYM$ is required to be CCA2-secure and key-private \cite{BBDP01AC}. \\
 \end{definitiontoc}
 
+\begin{definitiontoc}{Authenticated Hybrid Public Key Encryption Scheme}
+    An \emph{authenticated hybrid public encryption scheme} $\aHPKE$ is an authenticated encryption scheme built off of an $\HPKE$ and a $\MAC$ used in the following way:
+    \[\aHPKE.\encrypt : \KA.\SecretKey \times \KA.\PublicKey \times \SYM.\Plaintext \to \AuthEncrypted\bra{\SYM.\Plaintext}\]
+    where $\AuthEncrypted$ is the encrypted note type:
+    \[\AuthEncrypted\bra{\SYM.\Plaintext} := \MAC.\Tag \times \Encrypted\bra{\SYM.\Plaintext}\]
+    and the tag is computed by applying the $\MAC$ onto the encrypted note:
+    \[\TAG := \MAC(\sk, \HPKE.\encrypt(\esk, \pk))\]
+\end{definitiontoc}
+
```

Comment:
No more MAC since we are using duplex sponge encryption for authenticated symmetric encryption which has its own system for authentication tags. See https://github.com/Manta-Network/manta-rs/blob/main/manta-crypto/src/permutation/duplex.rs for more details.